src/public/modules/forms/admin/views/edit-fields.client.modal.html

```diff
@@ -589,7 +593,83 @@ <h2 class="modal-title">
           </div>
         </div>
 
-        <div class="row" ng-show="vm.field.fieldType === 'email'"><br /></div>
+        <div class="row" ng-show="vm.field.fieldType === 'email'">
+          <br />
+          <div class="toggle-option">
+            <div
+              ng-class="{'toggle-option--disabled' : !vm.field.isVerifiable}"
+              class="col-xs-8 label-custom label-medium"
+            >
+              Restrict email domains<i
+                class="glyphicon glyphicon-question-sign"
+                uib-tooltip="e.g. @mom.gov.sg, @moe.gov.sg OTP verification needs to be activated first."
```

Comment:
Can we conditionally show tooltip, like when OTP verification is not toggled on, we show the full example like this, but with a newline:

```
uib-tooltip="e.g. @mom.gov.sg, @moe.gov.sg\nOTP verification needs to be activated first."
```

but when the OTP verification is already toggled on, we only show the example
```
uib-tooltip="e.g. @mom.gov.sg, @moe.gov.sg
```

Reply:
Can

Reply:
@karrui Will need you verify if the way I did `$watch` is appropriate. Thanks!

Reply:
Correct! But small nit: call `vm.tooltipText` `tooltipHtml` instead :P

Reply:
Of course, we can. Pushing soon